### DISCUSSION TITLE
Articulation root is added to the base link, not the root prim as it should  (Isaac Sim 2023.1.1)

### DISCUSSION THREAD

**Discussion by TKZome** [NVlabs/curobo#162] [Software Q&A] [answered]
The articulation root is added to the base_link of the robot. It should be added to the "/World/ur5e_robot" 
The same thing happen when i run the standard "examples/isaac_sim/motion_gen_reacher.py" with the franka robot. 

![image](https://github.com/NVlabs/curobo/assets/60672398/ca03fa17-9b4b-485f-be6d-df81c401f22a)

I can't find the code where this happens, everything seems correct

**Comment by balakumar-s** [comment, marked as answer]
We do not have our examples working in Isaac sim 2023.1.1. Try an older version.
